lib/rules/no-action-modifiers.js

```diff
@@ -33,39 +33,22 @@ module.exports = class NoActionModifiers extends Rule {
   }
 
   visitor() {
-    this._element = null;
```

Comment:
These changes seem unrelated to the rule being added, please revert them (possibly separating to another PR if applicable)